app/helpers/context_helper.rb

```diff
@@ -23,6 +23,17 @@ module ContextHelper
     voters_count: { 'toot' => 'http://joinmastodon.org/ns#', 'votersCount' => 'toot:votersCount' },
     olm: { 'toot' => 'http://joinmastodon.org/ns#', 'Device' => 'toot:Device', 'Ed25519Signature' => 'toot:Ed25519Signature', 'Ed25519Key' => 'toot:Ed25519Key', 'Curve25519Key' => 'toot:Curve25519Key', 'EncryptedMessage' => 'toot:EncryptedMessage', 'publicKeyBase64' => 'toot:publicKeyBase64', 'deviceId' => 'toot:deviceId', 'claim' => { '@type' => '@id', '@id' => 'toot:claim' }, 'fingerprintKey' => { '@type' => '@id', '@id' => 'toot:fingerprintKey' }, 'identityKey' => { '@type' => '@id', '@id' => 'toot:identityKey' }, 'devices' => { '@type' => '@id', '@id' => 'toot:devices' }, 'messageFranking' => 'toot:messageFranking', 'messageType' => 'toot:messageType', 'cipherText' => 'toot:cipherText' },
     suspended: { 'toot' => 'http://joinmastodon.org/ns#', 'suspended' => 'toot:suspended' },
+    groups: {
+      'sm' => 'http://smithereen.software/ns#',
+      'litepub' => 'http://litepub.social/ns#',
+      'toot' => 'http://joinmastodon.org/ns#',
+      'wall' => { '@id' => 'sm:wall', '@type' => '@id' },
+      'members' => { '@id' => 'sm:members', '@type' => '@id' },
+      'capabilities' => 'litepub:capabilities',
+      'acceptsJoins' => 'litepub:acceptsJoins',
```

Comment:
That was needed for compatibility with Smithereen (which we don't have at the moment, since we purposefully use different addressing and actor type for actors, so to not have group posts end up in current Mastodon's UI and database). This is something we can drop altogether if we stick with `toot:PublicGroup`.